contracts/exchange/api/ConfigContract.sol

```diff
@@ -59,43 +59,52 @@ contract ConfigContract is BaseContract {
   ///////////////////////////////////////////////////////////////////
   /// Config Accessors
   ///////////////////////////////////////////////////////////////////
+
+  // unsafe casting here is expected, as the byte32 value represents an signed integer
   function _configToInt(bytes32 v) internal pure returns (int64) {
-    return int64(uint64(uint256(v)));
+    return int64(SafeCast.toUint64(uint256(v)));
```

Comment:
shouldn't this bee `SafeCast.toInt64(SafeCast.toInt256(uint256(v))))`

Reply:
No. We encode int with only the lower 8 bytes of the `bytes32` value. If we do `SafeCast.toInt64(SafeCast.toInt256(uint256(v))))`, `SafeCast.toInt64` would overflow for negative `int64` values(highest bit set to 1).

Reply:
The centiBeep config is a better example
```
  // unsafe casting here is expected, as the byte32 value represents an signed integer
  function _centiBeepToConfig(int32 v) internal pure returns (bytes32) {
    return bytes32(uint256(uint32(v)));
  }

  // unsafe casting here is expected, as the byte32 value represents an signed integer
  function _configToCentibeep(bytes32 v) internal pure returns (int32) {
    return int32(SafeCast.toUint32(uint256(v)));
  }
```

Reply:
ah that makes sense!